man/git-squash.md

```diff
@@ -20,17 +19,18 @@ git-squash(1) -- Import changes from a branch
   A commit reference (has to be from the current branch) can also be used as the
   first argument. A range of commits <sha>..HEAD will be squashed.
 
-  &lt;--squash-msg&gt;
```

Comment:
Why change the option name?

Reply:
so it's in sync with the SYNOPSIS, but TBH: i have not tested which one is the correct one.

Reply:
@elonderin 
I think it would be better if you have tried.

Reply:
u are right